### PR TITLE
Hours address telephone

### PR DESCRIPTION
**Pull request by Aturberv**
Idk whats up with the spacing in BreweryInfo, it doesn't show up that way in subl

### CODE REVIEW

File: src/BreweryInfo/BreweryInfo.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const BreweryInfo = ({ info }) => {
+	return(
+		<div>
+			<span>
+        <div>
```
Comment (ryansydnor) [inline src/BreweryInfo/BreweryInfo.js]: whoa whats up with the spacing

File: src/BreweryInfo/BreweryInfo.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const BreweryInfo = ({ info }) => {
+	return(
+		<div>
+			<span>
```
Comment (ryansydnor) [inline src/BreweryInfo/BreweryInfo.js]: can you render a div inside of a span?

File: src/BreweryInfo/BreweryInfo.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```
Comment (ryansydnor) [inline src/BreweryInfo/BreweryInfo.js]: i think we should reuse this component in the brewerymarker popover.

File: src/BreweryInfo/BreweryInfo.js (hunk 1 above)
Comment (Aturberv) [inline src/BreweryInfo/BreweryInfo.js]: No clue, its not like this on my sublime

File: src/BreweryPage/BreweryPage.js [hunk 4]
```diff
@@ -29,7 +30,23 @@ const BreweryPage = (
...
+            <meta itemProp="streetAddress" content={brewery.brewInfo.address.streetAddress} />
+            <meta itemProp="addressLocality" content={brewery.brewInfo.address.city} />
+            <meta itemProp="addressRegion" content={brewery.brewInfo.address.state} />
+            <meta itemProp="postalCode" content={brewery.brewInfo.address.postal} />
+          </div>
+          <meta itemProp="telephone" content={brewery.brewInfo.phone} />
+          {
+            brewery.brewInfo.hours.map((day) =>
+              <div itemProp="openingHoursSpecification" itemType="http://schema.org/OpeningHoursSpecification">
+                  <link itemProp="dayOfWeek" href={`http://schema.org/${config.days[day.open.day]}`} />
```
Comment (ryansydnor) [inline src/BreweryPage/BreweryPage.js]: wat is a link

**Pull request merged**